docs/toolbox/gpu_operator.rst

```diff
@@ -9,20 +9,20 @@ Deployment
 
 .. code-block:: shell
 
-    toolbox/gpu-operator/deploy_from_operatorhub.sh [<version>] [<channel>]
-    toolbox/gpu-operator/undeploy_from_operatorhub.sh
+    ./run.py gpu_operator deploy_from_operatorhub [<version>] [<channel>]
```

Comment:
I think `version` and `channel` require a flag, `--version=` (that's what I've seen above in the CI entrypoint at least)